pegasus/sites.v3/code.org/public/educate/csp/hall-of-fame.md

```diff
@@ -0,0 +1,438 @@
+* HYDE LEADERSHIP CHARTER SCHOOL, BRONX - Kate Anderson
+* HYDE LEADERSHIP CHARTER SCHOOL, BRONX - Mr.H
+* HIGH SCHOOL OF TELECOMMUNICATIONS ARTS AND TECHNOLOGY, BROOKLYN - jsosa
+* HIGH SCHOOL OF TELECOMMUNICATIONS ARTS AND TECHNOLOGY, BROOKLYN - ysears
+* INFORMATION TECHNOLOGY HIGH SCHOOL, LONG ISLAND CITY - Mrs.Ramirez
+* JOHN ADAMS HIGH SCHOOL, OZONE PARK - Ms.B Lestz
+* MANHATTAN BRIDGES HIGH SCHOOL, NEW YORK - IALONS0
+* NYC LAB HIGH SCHOOL FOR COLLABORATIVE STUDIES, NEW YORK - Bradley (Brad) Ashley
+* RENAISSANCE CHARTER SCHOOL (THE), JACKSON HEIGHTS - Richard Doherty
+* ROSLYN HIGH SCHOOL, ROSLYN HEIGHTS - JDiPietro
+* SCHOOL FOR LEGAL STUDIES, BROOKLYN - Alicia
+* SLEEPY HOLLOW HIGH SCHOOL, SLEEPY HOLLOW - MrLarkin
+* UNIONDALE HIGH SCHOOL, UNIONDALE - Ms.Georges
+* WILLIAMSVILLE EAST HIGH SCHOOL, EAST AMHERST - Mr. DeLellis
+* WILLIAMSVILLE NORTH HIGH SCHOOL, WILLIAMSVILLE - JulieJohnson
+* YOUNG WOMEN'S LEADERSHIP SCHOOL OF BROOKLYN, BROOKLYN - MrAnderson
+
+## North Carolina
+* CEDAR RIDGE HIGH, HILLSBOROUGH - Danial Nijhout-Rowe
+* CHATHAM CENTRAL HIGH, BEAR CREEK - Newman
+* J. F. WEBB HIGH, OXFORD - Marva Burrell
+
+## Ohio
+* MCCLAIN HIGH SCHOOL, GREENFIELD - MrLuke17
+* OTTAWA-GLANDORF HIGH SCHOOL, OTTAWA - Cody Vennekotter
+* REYNOLDSBURG HIGH SCHOOL ESTEM, REYNOLDSBURG - Mrs. Trachsel
+* REYNOLDSBURG HIGH SCHOOL ESTEM, REYNOLDSBURG - Ms. Dye
+
+## Pennsylvania
+* CAMBRIDGE SPRINGS JSHS, CAMBRIDGE SPRINGS - Vickie Schultz
+* EASTON AREA HS, EASTON - Mrs. C
+* SCHUYLKILL VALLEY HS, LEESPORT - Bryan Binkley
+* SWENSON ARTS & TECHNOLOGY HS, PHILADELPHIA - Mr_Karpe
+
+## South Dakota
+* WASHINGTON HIGH SCHOOL - 01, SIOUX FALLS - Mark Emry
+
+## Texas
+* EASTWOOD ACADEMY, HOUSTON - Mr. Platt
+* HOUSTON MATH SCIENCE AND TECHNOLOGY CENTER, HOUSTON - Manoj
+* HOUSTON MATH SCIENCE AND TECHNOLOGY CENTER, HOUSTON - Dehua Lea
+* HOUSTON MATH SCIENCE AND TECHNOLOGY CENTER, HOUSTON - Aaliyah Floyd
+* HOUSTON MATH SCIENCE AND TECHNOLOGY CENTER, HOUSTON - Ms. I. Perkins
+* LANCASTER H S, LANCASTER - Tracy Waller
+* LEE H S, HOUSTON - Barani
+* MICKEY LELAND COLLEGE PREP ACAD FOR YOUNG MEN, HOUSTON - plaforet
+* NORTH HOUSTON EARLY COLLEGE H S, HOUSTON - Fabiola Stroud
+* SCARBOROUGH H S, HOUSTON - amunoz2
+* STERLING H S, HOUSTON - jkendal2
+* VICTORIA WEST H S, VICTORIA - Mr.ODonnell
+* WESTBURY H S, HOUSTON - Maria Montero
+* YATES H S, HOUSTON - Wendy D. Neuls
+* YOUNG WOMEN'S COLLEGE PREP ACADEMY, HOUSTON - Astra Zeno
+
+## Utah
+* ACADEMY FOR MATH ENGINEERING & SCIENCE (AMES), SALT LAKE CITY - Vaughn Webster
+* COPPER HILLS HIGH, WEST JORDAN - Brandon Jacobson
+* COPPER HILLS HIGH, WEST JORDAN - Joel Marquez
+* ITINERIS EARLY COLLEGE HIGH, WEST JORDAN - Lynette Yorgason
+* JUAB HIGH, NEPHI - Tana Esplin
+* OLYMPUS HIGH, SALT LAKE CITY - Milton Watts
+* PINE VIEW HIGH, ST GEORGE - Zack Alvey
+* VENTURE ACADEMY, MARRIOTT-SLATERVILLE CITY - Bontia
+
+## Virginia
+* ALBEMARLE HIGH, CHARLOTTESVILLE - bholley@k12albemarle.org
```

Comment:
Is it okay to put people's emails on the site like this?

Reply:
Oops! Good catch! Looks like they provided their email when they should have provided their name. Fixing...